notebooks/233-blip-visual-language-processing/233-blip-visual-language-processing.ipynb

```diff
@@ -143,7 +143,7 @@
     }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.                placeholder="If this field is empty, an image caption will be generated",
</pre><p>placeholder visible only when initially empty string, if user write something and then remove, it will not visible anymore. Could you please add some description for text box (using markdown in interface or just before cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
On my end, it does render when the field is empty in all cases. Added `info` instead of `placeholder`.